ui.tests/test-module/specs/customization/customtextinput.runtime.spec.js

```diff
@@ -64,7 +64,7 @@ describe("Form Runtime with Custom Text Input", () => {
         expect(formContainer, "formcontainer is initialized").to.not.be.null;
         expect(formContainer._model.items.length, "model and view elements match").to.equal(Object.keys(formContainer._fields).length);
         Object.entries(formContainer._fields).forEach(([id, field]) => {
-            if(field.options.visible === "true") {
+            if(field.options.visible === "true" && field.getModel()._jsonModel.fieldType==='text-input') {
```

Comment:
Why is there a change in an existing test case ? This can impact backward compatibility

Reply:
Attributes 'data-cmp-enabled' and 'data-cmp-visible' are causing button to be present always in the form which is why the checkHTML will fail if we do not explicitly check for 'text-input'. I have confirmed all components in both textinput and customtextinput test files have all components with fieldType 'text-input' apart from a submit button.